frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -265,50 +265,45 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
 
   getLayerSettings = (
     layerName: string,
-    layer: ?DatasetLayerConfiguration,
+    layerConfiguration: ?DatasetLayerConfiguration,
     isColorLayer: boolean = true,
   ) => {
-    // Ensure that a color layer needs a layer.
-    if (isColorLayer && !layer) {
+    // Ensure that a every layer needs a layer configuration.
+    if (!layerConfiguration) {
       return null;
     }
     const elementClass = getElementClass(this.props.dataset, layerName);
     const isDisabled =
-      isColorLayer && layer != null
-        ? layer.isDisabled
+      isColorLayer && layerConfiguration != null
```

Comment:
Why is this check necessary? In line 272 you already return from this function if the var is null?

Reply:
Nice thanks for the catch :eyes: